modelplugin/aether-2.1.0/aether-2.1.0.tree

```diff
@@ -0,0 +1,152 @@
+        +--rw enabled?                 boolean
+        +--rw enterprise?              -> /ent:enterprise/enterprise/id
+        +--rw profiles
+           +--rw apn-profile?        -> /apn:apn-profile/apn-profile/id
+           +--rw up-profile?         -> /up:up-profile/up-profile/id
+           +--rw qos-profile?        -> /qos:qos-profile/qos-profile/id
+           +--rw access-profile* [access-profile]
+           |  +--rw access-profile    -> /ap:access-profile/access-profile/id
+           |  +--rw allowed?          boolean
+           +--rw security-profile?   -> /sec:security-profile/security-profile/id
+
+module: apn-profile
+  +--rw apn-profile
+     +--rw apn-profile* [id]
+        +--rw id               apn-profile-id
+        +--rw display-name?    string
+        +--rw apn-name?        string
+        +--rw dns-primary?     inet:ip-address
+        +--rw dns-secondary?   inet:ip-address
+        +--rw mtu?             uint32
+        +--rw gx-enabled?      boolean
+        +--rw description?     string
+        +--rw service-group?   -> /sg:service-group/service-group/id
+
+module: connectivity-service
+  +--rw connectivity-service
+     +--rw connectivity-service* [id]
+        +--rw id                connectivity-service-id
+        +--rw display-name?     string
+        +--rw description?      string
+        +--rw spgwc-endpoint?   inet:uri
+        +--rw hss-endpoint?     inet:uri
+
+module: enterprise
+  +--rw enterprise
+     +--rw enterprise* [id]
+        +--rw id                      enterprise-id
+        +--rw display-name?           string
+        +--rw description?            string
+        +--rw connectivity-service* [connectivity-service]
+           +--rw connectivity-service    -> /cs:connectivity-service/connectivity-service/id
+           +--rw enabled?                boolean
+
+module: qos-profile
+  +--rw qos-profile
+     +--rw qos-profile* [id]
+        +--rw id              qos-profile-id
+        +--rw display-name?   string
+        +--rw apn-ambr
+        |  +--rw uplink?     uint32
+        |  +--rw downlink?   uint32
+        +--rw qci?            uint32
+        +--rw arp
+        |  +--rw priority?                   uint32
+        |  +--rw preemption-capability?      boolean
+        |  +--rw preemption-vulnerability?   boolean
+        +--rw description?    string
+
+module: security-profile
+  +--rw security-profile
+     +--rw security-profile* [id]
+        +--rw id              security-profile-id
+        +--rw display-name?   string
+        +--rw key?            string
+        +--rw opc?            string
+        +--rw sqn?            uint32
+        +--rw description?    string
+
+module: service-group
+  +--rw service-group
+     +--rw service-group* [id]
+        +--rw id              service-group-id
+        +--rw display-name?   string
+        +--rw services* [service]
+        |  +--rw service    -> /serv:service-policy/service-policy/id
```

Comment:
maybe the attribute should be `service-policy` - maybe the list should be `service-policies`

Reply:
This was because I had to rename "service" to "service-policy" to avoid the prefix-matching bug (which I still need to ticket). Let me think on whether I want that to be a temporary fix, or want service-policy to be permanent.